.pre-commit-config.yaml

```
@@ -0,0 +1,25 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.1.0
+    hooks:
+    - id: check-toml
+    - id: end-of-file-fixer
+    - id: trailing-whitespace
+  - repo: https://github.com/PyCQA/isort
+    rev: 5.10.1
+    hooks:
+    - id: isort
+  - repo: https://github.com/psf/black
+    rev: 22.1.0
```

Fixes issue with `click`.

```
ImportError: cannot import name '_unicodefun' from 'click'
```

```suggestion
    rev: 22.3.0
```